irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -284,5 +306,23 @@ namespace iroha {
       }
       return true;
     }
+
+    bool PostgresWsvCommand::setAccountKV(const std::string &account_id,
+                                          const std::string &key,
+                                          const std::string &val) {
+      try {
+        transaction_.exec(
+            "UPDATE account\n"
+            "   SET data = jsonb_set(data,\'{"+key+"}\', \'\""+val+"\"\')"
+           "\n WHERE account_id="
+            + transaction_.quote(account_id)
+            + ";");
+      } catch (const std::exception &e) {
```

Comment:
Maybe use [raw string literals](https://solarianprogrammer.com/2011/10/16/cpp-11-raw-strings-literals-tutorial/) for jsonb?